src/modules/dna-service/dna.background.ts

```diff
@@ -9,8 +9,11 @@ import * as browserStorage from 'src/background/webapis/storage';
 import { normalizeAddress } from 'src/shared/normalizeAddress';
 import { emitter } from 'src/background/events';
 import { isReadonlyContainer } from 'src/shared/types/validators';
+import type { Wallet } from 'src/shared/types/Wallet';
+import { INTERNAL_SYMBOL_CONTEXT } from 'src/background/Wallet/Wallet';
 import type { DnaAction } from './types';
 
+const REGISTER_ALL_WALLETS_INVOKED_KEY = 'registerAllWalletsInvoked-14-10-2024';
```

Comment:
very minor but i think if the key has hyphens it's more readable to have the whole value like that

Reply:
good point, I'll update the key

Reply:
I see, it mirrors the method name. In that case perhaps it's okay the way it is

Reply:
Ok, if it reads well for you then I leave it as is